scripts/setup-jest.ts

```diff
@@ -64,7 +67,11 @@ const getMockVSCode = () => {
         };
       },
       onDidChangeConfiguration: jest.fn(),
-      createFileSystemWatcher: jest.fn(),
+      createFileSystemWatcher: jest.fn().mockReturnValue({
+        onDidChange: jest.fn(),
+        onDidCreate: jest.fn(),
+        onDidDelete: jest.fn()
+      }),
```

Comment:
new vscode mock setup to get things working for core.  Note that b/c the src/context/index.ts creates an instance of WorkspaceContext on initialization
 `export const workspaceContext = WorkspaceContext.getInstance();`
We need to mock the file watcher results prior to loading any test dependancies.  
This is an anti-pattern we should avoid for singletons.